nservicebus/azure/index.md

```diff
@@ -3,5 +3,23 @@ title: NServiceBus and Azure
 summary: Using Azure for endpoint hosting and to provide Transports and Persistence
 tags:
 - Azure
-reviewed: 2016-09-21
+reviewed: 2017-03-28
 ---
+
+NServiceBus helps create distributed .NET systems not only on premises but also in a public cloud environment. That is why it provides a variety of capabilities built for Microsoft Azure, the cloud closest to .NET. This covers hosting, Azure specific transports and persistence.
```

Comment:
Still on the fence because it diminishes other clouds. How about

> NServiceBus helps create distributed .NET systems not only on premises but also in the Microsoft Azure cloud. It provides a variety of capabilities built for Microsoft Azure such as hosting, Azure specific transports and persistence.